src/js/tabs/osd.js

```diff
@@ -306,6 +306,22 @@ OSD.generateAltitudePreview = function(osdData) {
     return `${FONT.symbol(SYM.ALTITUDE)}399${variantSelected === 0? '.7' : ''}${unit}`;
 };
 
+
+OSD.generateVTXChannelPreview = function(osdData) {
+    const variantSelected = OSD.getVariantForPreview(osdData, 'VTX_CHANNEL');
+    let value;
+    switch (variantSelected) {
+        case 0:
+            value = 'R:2:200:P';
+            break;
+
+        case 1:
+            value = 'PWR';
```

Comment:
probably instead of PWR it should show, for example: "250MW" or whatever format is correct.

Reply:
I have it as "PWR" because the displayed value is just a number without a unit to keep the OSD element as short as possible. I can change to "200". 

thoughts?

Reply:
ok yeah i think in this case its ok

Reply:
We usually use one of the "real" representations in the OSD. If it is simply a number, use a number.